system/default_ad_api/launch/default_ad_api.launch.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2022 TIER IV, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import launch
+from launch_ros.actions import ComposableNodeContainer
+from launch_ros.actions import Node
+from launch_ros.descriptions import ComposableNode
+
+
+def _create_api_node(node_name, class_name, **kwargs):
+    return ComposableNode(
+        namespace="default_ad_api/node",
+        name=node_name,
+        package="default_ad_api",
+        plugin="default_ad_api::" + class_name,
+        **kwargs
+    )
+
+
+def generate_launch_description():
+    components = [
+        _create_api_node("interface_version", "InterfaceVersionNode"),
+    ]
+    container = ComposableNodeContainer(
+        namespace="default_ad_api",
+        name="container",
+        package="rclcpp_components",
+        executable="component_container_mt",
+        composable_node_descriptions=components,
+    )
+    server = Node(
+        package="default_ad_api",
+        name="server",
+        executable="server.py",
+    )
+    return launch.LaunchDescription([container, server])
```

Comment:
I think it's good to add `web` here as well.

```suggestion
    web_server = Node(
        package="default_ad_api",
        name="web_server",
        executable="web_server.py",
    )
    return launch.LaunchDescription([container, web_server])
```